src/kbmod/filters/stamp_filters.py

```diff
@@ -201,26 +252,37 @@ def filter_stamps_by_cnn(result_data, model_path, coadd_type="mean", stamp_radiu
     verbose : `bool`
         Verbosity option for the CNN predicition. Off by default.
     """
-    from tensorflow.keras.models import load_model
 
     coadd_column = f"coadd_{coadd_type}"
     if coadd_column not in result_data.colnames:
         raise ValueError("result_data does not have provided coadd type as a column.")
 
-    cnn = load_model(model_path)
-
     stamps = result_data.table[coadd_column].data
     stamp_dimm = (stamp_radius * 2) + 1
+    stamp_shape = (1, stamp_dimm, stamp_dimm)
     normalized_stamps = _normalize_stamps(stamps, stamp_dimm)
-
-    # resize to match the tensorflow input
-    # will probably not be needed when we switch to PyTorch
-    resized_stamps = normalized_stamps.reshape(-1, stamp_dimm, stamp_dimm, 1)
-
-    predictions = cnn.predict(resized_stamps, verbose=verbose)
+    normalized_stamps = np.expand_dims(normalized_stamps, axis=1)
+
+    model = MODEL_TYPES[model_type](num_classes=2)
+    cnn = _KBMLModel(
+        model=model,
+        weights=model_path,
+        shape=stamp_shape,
+    )
+    # we'll need to check if we have a GPU available.
+    if model_path:
+        if torch.cuda.is_available():
+            cnn.load_state_dict(torch.load(model_path))
+        else:
+            cnn.load_state_dict(torch.load(model_path, map_location=torch.device("cpu")))
+    cnn.eval()
```

Comment:
Can you add a comment here that this just sets the model to evaluation model? I was originally confused, because I thought it was doing the computation here.